qemu/tests/qemu_guest_agent.py

```diff
@@ -1580,14 +1605,9 @@ def gagent_check_file_read(self, test, params, env):
         # if guest os resource is enough, will return no error.
         # else it will return error like "insufficient system resource"
         # which is expected
-        try:
-            self.gagent.guest_file_read(ret_handle, count=1000000000)
-        except guest_agent.VAgentCmdError as detail:
-            info_insuffi = "Insufficient system resources exist to"
-            info_insuffi += " complete the requested service"
-            if not re.search(info_insuffi, str(detail)):
-                test.fail("Return error but is not the desired information: "
-                          "('%s')" % str(detail))
+        count = 1000000000 if params["os_type"] == 'linux' and \
+                main_qga_ver<5 else 10000000
```

Comment:
Better avoid `\`, and need space before and after `<`(guess that's how CI check failed), you might write it like this:
```
count = 1000000000 if (params["os_type"] == 'linux' and
                       main_qga_ver < 5) else 10000000
```

Reply:
okay,thanks, updating it

Reply:
the reason of CI check failed seems a little weird, something doesn't go wrong : /home/travis/build/autotest/tp-qemu/qemu/tests/qemu_guest_agent.py:1645:64: W291 trailing whitespace, but the line has nothing wrong.

Reply:
@6-dehan That's a quite common issue, it means you have an extra wthitespace at the end of the line, which is not allowed and it would be fine after you remove it.

Reply:
I met the issue you said, but it's not the same issue this time, the line 1645 and other lines around it I've checked and even re-edit them refer to unmodified content from master branch.but issue still be there.really weird.